src/main/java/org/jenkinsci/maven/plugins/hpi/HpiMojo.java

```diff
@@ -149,15 +149,15 @@ private void performPackaging()
                          final OutputStreamWriter indexJellyWriter = new OutputStreamWriter(fos, StandardCharsets.UTF_8)) {
                         indexJellyWriter.write("<?jelly escape-by-default='true'?>\n" +
```

Comment:
This should be setting an XML encoding line.

```xml
<?xml version="1.0" encoding="UTF-8"?>
```

Reply:
The previous error suggested:

```
throw new MojoFailureException("Missing " + indexJelly + ". Delete any <description> from pom.xml and create src/main/resources/index.jelly:\n" +
                    "<?jelly escape-by-default='true'?>\n" +
                    "<div>\n" +
                    "    The description here…\n" +
                    "</div>");
```
🤣

Reply:
Fixed it